app/javascript/mastodon/features/audio/index.js

```diff
@@ -103,7 +103,7 @@ class Audio extends React.PureComponent {
   }
 
   componentDidUpdate (prevProps, prevState) {
-    if (prevProps.src !== this.props.src || this.state.width !== prevState.width || this.state.height !== prevState.height) {
+    if (prevProps.src !== this.props.src || this.state.width !== prevState.width || this.state.height !== prevState.height || prevProps.accentColor !== this.props.accentColor) {
```

Comment:
Hmm I suppose this needs take background and foreground colors into account too.

Reply:
Nope, those are properly passed down, and not part of the canvas.

Reply:
Wait, we should just make a `poster` check. The colors can't change unless the poster image changed.

Reply:
Oh right, only accentColor gets *drawn* currently. I see what you mean now. Yeah this makes sense then.